test/support/membrane_element_fdk_aac/encoding_pipeline.ex

```diff
@@ -1,55 +1,57 @@
 defmodule Membrane.AAC.FDK.Support.EncodingPipeline do
   @moduledoc false
 
+  import Membrane.ChildrenSpec
+
   alias Membrane.Testing.Pipeline
 
-  defmodule CapsProvider do
+  defmodule StreamFormatProvider do
     @moduledoc false
     use Membrane.Filter
 
-    def_options input_caps: [
+    def_options input_stream_format: [
                   description:
-                    "Caps which will be sent on the :output pad once the :input pad receives any caps",
-                  type: :caps
+                    "stream_format which will be sent on the :output pad once the :input pad receives any stream_format",
+                  type: :stream_format
                 ]
 
-    def_output_pad :output, demand_mode: :auto, caps: :any
+    def_output_pad :output, demand_mode: :auto, accepted_format: _any
 
-    def_input_pad :input, demand_unit: :bytes, demand_mode: :auto, caps: :any
+    def_input_pad :input, demand_unit: :bytes, demand_mode: :auto, accepted_format: _any
 
     @impl true
-    def handle_init(opts) do
-      {:ok, %{caps: opts.input_caps}}
+    def handle_init(_ctx, opts) do
+      {[], %{stream_format: opts.input_stream_format}}
     end
 
     @impl true
-    def handle_caps(:input, _caps, _ctx, state) do
-      {{:ok, caps: {:output, state.caps}}, state}
+    def handle_stream_format(:input, _stream_format, _ctx, state) do
+      {[stream_format: {:output, state.stream_format}], state}
     end
 
     @impl true
     def handle_process(:input, buffer, _ctx, state) do
-      {{:ok, buffer: {:output, buffer}}, state}
+      {[buffer: {:output, buffer}], state}
     end
   end
 
   @spec make_pipeline(Path.t(), Path.t()) :: GenServer.on_start()
   def make_pipeline(in_path, out_path) do
-    Pipeline.start_link(%Pipeline.Options{
-      elements: [
-        file_src: %Membrane.File.Source{location: in_path},
-        caps_provider: %CapsProvider{
-          input_caps: %Membrane.RawAudio{
-            sample_format: :s16le,
-            channels: 2,
-            sample_rate: 44_100
-          }
-        },
-        encoder: %Membrane.AAC.FDK.Encoder{
-          input_caps: %{sample_rate: 44_100, channels: 2}
-        },
-        sink: %Membrane.File.Sink{location: out_path}
-      ]
-    })
+    structure = [
+      child(:file_src, %Membrane.File.Source{location: in_path})
+      |> child(:stream_format_provider, %StreamFormatProvider{
+        input_stream_format: %Membrane.RawAudio{
+          sample_format: :s16le,
+          channels: 2,
+          sample_rate: 44_100
+        }
+      })
+      |> child(:encoder, %Membrane.AAC.FDK.Encoder{
+        input_stream_format: %{sample_rate: 44_100, channels: 2}
+      })
+      |> child(:sink, %Membrane.File.Sink{location: out_path})
+    ]
+
+    Pipeline.start_link(structure: structure)
```

Comment:
Since `start_link` now returns tuple of three elements, `make_pipeline` no more returns `GenServer.on_start()`, as the typespec says